api_app/serializers.py

```diff
@@ -169,7 +169,7 @@ def filter_analyzers(self, serialized_data: Dict) -> List[str]:
                         f"{a_name} won't run: is disabled or unconfigured"
                     )
 
-                if tlp != TLP.WHITE and config.leaks_info:
+                if tlp != TLP.CLEAR and config.leaks_info:
```

Comment:
I would like to keep WHITE too for retrocompatibility with previous IntelOwl versions. Because of this it could make sense to add a class method to the TLP class which retrieves both the TLP.CLEAR and the TLP.WHITE.

This would be the case where that classmethod could be used

Reply:
Please let me know if the updated code addresses this comment. If so, wouldn't it be easier to understand if we used `in` with the tuple `(TLP.WHITE, TLP.CLEAR)` when checking for `TLP.CLEAR`? And in the api_app/views.py file, list both `TLP.WHITE` and `TLP.CLEAR` along with `TLP.GREEN`.